libraries/chain/include/eosio/chain/host_context.hpp

```diff
@@ -599,6 +601,9 @@ class host_context {
    // act pointer may be invalidated on call to trx_context.schedule_action
    iterator_cache<key_value_object>    keyval_cache;
    std::string                         _pending_console_output;
+
+   bool                                _is_action    = true;  // used to avoid using dynamic_cast to identify type of host_context
+   bool                                _is_sync_call = false; // used to avoid using dynamic_cast to identify type of host_context
```

Comment:
These can be `const`.

Reply:
They were removed by https://github.com/AntelopeIO/spring/pull/1296/commits/0b3559cc857cfd90dc735c4b76be489b81410a92